ccswbs/hjckrrh
# Issue 806 - Fix Accordion Script

Accordion script currently looks for `data-target` attributes on `<a class="collapse">` links and automatically sets the `href` of the element to match the target. It was setting the value incorrectly though, which caused the accordion controller to navigate to another page.

This pull request removes the line altogether because a controller using the `<a>` tag only requires an `href` attribute as a fallback for no JavaScript, so there is no point is setting the attribute with JavaScript.

## Manual Test Procedure
- Create a page with an accordion structure (see https://aoda-test.web.uoguelph.ca/ccs/accordion-test for an example and https://github.com/ccswbs/hjckrrh/pull/735 for more information)
- Create an `<a>` controller with both `data-target` and `href` set
- Create an `<a>` controller with only `data-target` set (note that this is not ideal because it has no fallback, but it should still work)
- Ensure both the controllers properly expand/collapse their respective sections
